sale_fixed_discount/models/sale_order.py

```diff
@@ -39,6 +39,7 @@ class SaleOrderLine(models.Model):
 
     @api.onchange('discount')
     def _onchange_discount(self):
+        super(SaleOrderLine, self)._onchange_discount()
```

Comment:
`res = super(...)`

and return that value at the end (it can contain a domain)

Reply:
Are you sure?

Reply:
done!!..